collector/collector.go

```diff
@@ -83,7 +84,7 @@ func findMaps(key monitors.MetricName, maps ...map[monitors.MetricName]monitors.
 	return nil, false
 }
 
-func (c *LCDCollector) RegisterMonitor(ctx context.Context, m monitors.Monitor) {
+func (c *LCDCollector) RegisterMonitor(cfg config.CollectorConfig, ctx context.Context, m monitors.Monitor) {
```

Comment:
According to Context documentation, context argument should be the first one.

Reply:
Will fix.